src/lib/components/compositions/radio/animated.svelte

```diff
@@ -5,15 +5,16 @@
   import type { Maybe } from '$lib/interfaces';
   import Label from '$components/primitives/label/label.svelte';
 
-  export let defaultOption: number;
+  export let defaultOption: string | null = null;
   export let message = '';
   export let firstInvalidInput: boolean;
+  export let groupMessage: string;
```

Comment:
To keep the naming consistent with the rest of the components, I think calling this a `groupLabel` would be better than `groupMessage`. I say that since labels (as a convention in this project) generally go above the input and messages below.

Reply:
Makes sense to me. Will do.